src/shared/views/OrgsView.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { push } from 'connected-react-router';
-import { Button, Modal, Drawer, notification } from 'antd';
-import { OrgResponseCommon } from '@bbp/nexus-sdk';
+import { Button, Modal, Drawer, notification, message } from 'antd';
+import { OrgResponseCommon, Resource } from '@bbp/nexus-sdk';
 import { AccessControl, useNexusContext } from '@bbp/react-nexus';
 
 import OrgList from '../containers/OrgList';
 import OrgForm from '../components/Orgs/OrgForm';
 import OrgItem from '../components/Orgs/OrgItem';
 import ListItem from '../components/List/Item';
+import useQueryString from '../hooks/useQueryString';
+import { useHistory } from 'react-router';
+import { matchResultUrls, parseProjectUrl } from '../utils';
```

Comment:
Looks like matchResultUrls is an unused import?

Reply:
yes I'll get rid of it